### DISCUSSION TITLE
Forge Fluids Converter

### DISCUSSION THREAD

**Discussion by TheVitya2127** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1186] [Q&A]
The question is, **is it possible to convert liquids from ntm to other mods?**
And is it even possible to do this?

**Comment by MetalloloM-Edition** [comment]
#BobHelp

**Comment by BallOfEnergy1** [comment]
I mean, its possible (not converting fluids, but making NTM use forge fluids instead of bobfluids), but it would be tedious and the system we have right now works better than any system that could be put in place using forge fluids.

**Reply by TheVitya2127** [reply]
Yes, but because of this, compatibility with many mods breaks. For example: Railcraft, Traincraft

**Comment by TheVitya2127** [comment]
Now the question is, how do I combine these mods for my build? The bob will not make compatibility with this mod, and I am 99.9% sure of this

**Comment by Antineutrino-4444** [comment]
There are mods that does so, but are all broken currently

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by TheVitya2127** [reply]
I have windows...

**Comment by sdddddf80** [comment]
I guess you are looking for this mod:https://gitee.com/polonium210/HBMAdder. The brief instructions for its use are here:https://www.mcmod.cn/class/7426.html
